include/libtorrent/torrent_status.hpp

```diff
@@ -468,6 +469,16 @@ namespace libtorrent
 		// true when the torrent is in sequential download mode. In this mode
 		// pieces are downloaded in order rather than rarest first.
 		bool sequential_download;
+#else
+		// hidden
+		bool _dummy_ip_filter_applies;
```

Comment:
in the master branch, there's a macro ``TORRENT_DEPRECATED_MEMBER`` to annotate members as deprecated.